### PR TITLE
Add sort.callback.description entity

### PR DESCRIPTION
**Pull request by koseki**
* #2365

### CODE REVIEW

**Review by Girgias — CHANGES_REQUESTED** [review]
Please see related issue for requested changes.

File: language-snippets.ent [hunk 1]
```diff
@@ -692,6 +692,20 @@ use.</simpara></note>'>
...
+ &return.callbacksort;
+</para>
+&callback.cmp;
+<caution xmlns="http://docbook.org/ns/docbook">
+ <para>
+  Returning <emphasis>non-integer</emphasis> values from the comparison
+  function, such as <type>float</type>, will result in an internal cast to
+  <type>int</type> of the callback&apos;s return value. So values such as
+  0.99 and 0.1 will both be cast to an integer value of 0, which will
+  compare such values as equal.
```
Comment (Girgias) [inline language-snippets.ent:705]:
```suggestion
  <literal>0.99</literal> and <literal>0.1</literal> will both be cast to an
  integer value of <literal>0</literal>, which will compare such values as equal.
```

**Review by Girgias — APPROVED** [review]
Small nit about adding literal tags but LGTM :)

**Comment by koseki** [comment]
@Girgias Thank you for reviewing!

There seems to be no reference to the `&return.callbacksort;` other than here.
Can I merge this into `&sort.callback.description;` or should be left at this moment?

**Pull request merged**